Modules/Core/TestKernel/src/itkTestDriverIncludeRequiredFactories.cxx

```diff
@@ -41,12 +41,42 @@
 #include "itkOFFMeshIOFactory.h"
 #include "itkVTKPolyDataMeshIOFactory.h"
 
+// FFT
```

Comment:
Doesn't `itkTestDriverIncludeRequiredFactories.h` already include all of these? If so, the above list of IOs should be cleaned up too.

Reply:
`itkTestDriverIncludeBuildInFactories.h` includes these while `itkTestDriverIncludeRequiredFactories.h` only declares the functions that are implemented in `itkTestDriverIncludeRequiredFactories.cxx`. Ideally we should keep factory registrations in the `.cxx` implementation file.